### PR TITLE
Remove aclk_api.[ch]

### PR DESCRIPTION
**Pull request by underhood**
##### Summary
Main purpose of this file was to serve as a switch and common API between ACLK-NG and ACLK-Legacy. This has lost meaning after removal of legacy.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: aclk/aclk.c [hunk 1]
```diff
@@ -1204,3 +1214,38 @@ char *ng_aclk_state_json(void)
     json_object_put(msg);
     return str;
 }
+
+void add_aclk_host_labels(void) {
+    DICTIONARY *labels = localhost->host_labels;
+
+#ifdef ENABLE_ACLK
+    rrdlabels_add(labels, "_aclk_ng_available", "true", RRDLABEL_SRC_AUTO|RRDLABEL_SRC_ACLK);
```
Comment (thiagoftsm) [inline aclk/aclk.c:1222]: Is there a special motive to have this call for `rrdlabels_add` here instead next group of `ifdef ENABLE_ACLK`?

Comment (thiagoftsm) [reply aclk/aclk.c:1222]: Considering that both calls will change the third argument only, I suggest to use the `ifdef` to define only it.

Comment (underhood) [reply aclk/aclk.c:1222]: thanks I changed it. I only moved this function from file to file without checking the code.

**Review by thiagoftsm — APPROVED** [review]
I did not have any error during compilation time and cloud is working as expected, LGTM!

**Review by thiagoftsm — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**